### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by aju**
closes https://github.com/brainly/style-guide/issues/474

### CODE REVIEW

**Comment by mr-mig** [comment]
:ok:

File: src/components/text/links.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        Fine print link <a href="#" class="mint-link mint-link--for-fine-print-light">zadane.pl</a><br/>
+      </div>
+      <div>
+        Fine print emphasised link <a href="#" class="mint-link mint-link--for-fine-print mint-link--emphasised">Terms of use</a><br/>
+      </div>
+      <div>
+        Example of <a href="#" class="mint-link mint-link--error"> error link</a> and <a href="#" class="mint-link mint-link--warning"> warning link</a><br/>
+      </div>
+      <div>
+        Some texts and a <a href="#" class="mint-link mint-link--underlined">underlined link</a><br/>
```
Comment (kdzwinel) [inline src/components/text/links.html]: "some text" ;)

**Comment by kdzwinel** [comment]
Screenshot? Please :cat:

**Comment by aju** [comment]
TADA!
screen shot 2015-11-05 at 10 51 04

**Comment by kdzwinel** [comment]
OK

**Pull request merged**